products/brms/overview.adoc

```diff
@@ -41,8 +42,8 @@ image:#{cdn(site.base_url + '/images/icons/products/products_bar_graph.png')}["B
 == Detail
 
 [.large-12.columns]
-Business experts and application developers in organizations of any size need to be able to model, automate, measure, and improve their critical processes and policies. Red Hat(R) JBoss(R) BRMS makes it possible with fully integrated business rules management and complex event processing (CEP). All through an open source subscription. Red Hat(R) JBoss(R) BRMS includes powerful new tools for optimizing and scheduling business resources. Learn how the Business Resource Planner can tackle a wide range of planning problems, from delivery routing to shift scheduling and even managing compute resources in the cloud. +
+Learn how the Business Resource Planner can tackle a wide range of planning problems, from delivery routing to shift scheduling and even managing compute resources in the cloud. +
 Download JBoss BRMS and explore the tools for authoring business rules, provided as both standalone web applications and plug-ins for JBoss Developer Studio.
 
 [.large-12.columns]
-image:#{cdn(site.base_url + '/images/products/brms/capture.png')}["BRMS Screenshot", max-width=600]
+https://vimeo.com/album/2982444/video/143059283[image:#{cdn(site.base_url + '/images/products/brms/planner-video.png')}["Planner Video", max-width=600]]
```

Comment:
This should use the same code for adding videos that we have in other parts of the site:

    #{video_thumb(video('https://vimeo.com/143059283'), 'video_thumb.html.slim')}